Modules/Quotas/src/Quotas.Domain/Aggregates/Tiers/TierQuotaDefinition.cs

```diff
@@ -1,9 +1,10 @@
+using Backbone.BuildingBlocks.Domain;
 using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
-using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using MetricKey = Backbone.Modules.Quotas.Domain.Aggregates.Metrics.MetricKey;
```

Comment:
There are two `MetricKey` classes: One in `Backbone.BuildingBlocks.Domain` and the other in `Backbone.Modules.Quotas.Domain.Aggregates.Metrics`. Since I import both namespaces (bc `Entity` is in `Backbone.BuildingBlocks.Domain`), I have to specify which `MetricKey` to use.

Reply:
Ah, I see. Weird. I wonder why MetricKey is at the root level of the BuildingBlocks. But okay, we keep it as it is for now.